test/dispatcherTests.ts

```diff
@@ -1,28 +1,17 @@
 import 'jasmine';
-import * as actionCreator from '../src/actionCreator';
-import * as dispatcher from '../src/dispatcher';
-import * as globalContext from '../src/globalContext';
-
-describe('dispatcher', () => {
-    let mockGlobalContext: any;
-
-    beforeEach(() => {
-        mockGlobalContext = {
-            subscriptions: {},
-            dispatchWithMiddleware: jasmine.createSpy('dispatchWithMiddleware'),
-            currentMutator: null,
-        };
-
-        spyOn(globalContext, 'getGlobalContext').and.returnValue(mockGlobalContext);
-    });
+import { createTestSatchel } from './utils/createTestSatchel';
+import * as privateUtils from '../src/privatePropertyUtils';
 
+describe('dispatch', () => {
+    /*
     it('subscribe registers a callback for a given action', () => {
```

Comment:
Need to determine if this is covered by other tests or if it needs to be implemented.